code-challenges.test.js

```diff
@@ -10,42 +10,123 @@
 // Reminder: The test will call your function
 // Run the file with the following command: $ yarn jest
 
-// --------------------1) Create a function that takes in an array of objects and returns an array with a sentence about each person with their name capitalized.
 
+
+
+// describe(" ", () => {
+//       it("", () => {
+//         expect();
+//       });
+//   });
+
+
+
+
+
+
+// --------------------1) Create a function that takes in an array of objects and returns an array with a sentence about each person with their name capitalized.
 // a) Create a test with an expect statement using the variable provided.
 
+
+
+
+
+describe("personCapitalized", () => {
 const people = [
   { name: "ford prefect", occupation: "a hitchhiker" },
   { name: "zaphod beeblebrox", occupation: "president of the galaxy" },
-  { name: "arthur dent", occupation: "a radio employee" }
-]
-// Expected output: ["Ford Prefect is a hitchhiker.", "Zaphod Beeblebrox is president of the galaxy.", "Arthur Dent is a radio employee."]
+  { name: "arthur dent", occupation: "a radio employee" }]
+      // Expected output: ["Ford Prefect is a hitchhiker.", "Zaphod Beeblebrox is president of the galaxy.", "Arthur Dent is a radio employee."]
+    it("takes in an array of objects and returns an array with a sentence about each person with their name capitalized.", () => {
+      expect(personCapitalized(people)).toEqual(["Ford Prefect is a hitchhiker.", "Zaphod Beeblebrox is president of the galaxy.", "Arthur Dent is a radio employee."]);
+    });
+});
+
+
 
 
 // b) Create the function that makes the test pass.
 
 
+const personCapitalized = (array) => {
+  return array.map((object) => `${object.name.split(" ").map((word) => word.charAt(0).toUpperCase() + word.slice(1)).join(" ")} is ${object.occupation}.`
+  );
+};
```

Comment:
Your logic looks great. I would like to see some pseudo coding so I can understand how you are thinking through this problem.